backend/models/users.py

```diff
@@ -27,7 +26,7 @@ class User(db.Model):
     name = db.Column(db.Text)
     alias = db.Column(db.Text)
     email = db.Column(db.Text)
-    hashed_password = db.Column(db.Text, nullable=False)
```

Comment:
In what cases this field can be NULL?

Reply:
when we create a new user (in add user option) there's no password field

Reply:
so eventually we got an error, this can fix the error until we add a password field or whatever we're gonna do

Reply:
OK, got it. Let it be in such way.

Reply:
i find it just a temporary feature, cause we need to let users a possibility to create passwords or whatever in future